src/components/views/messages/CodeBlock.tsx

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
It was necessary to remove the GenericTextContextMenu which still used the `mx_Tooltip` styles. The old TextualBody consumer of that was rendering it out-of-React-tree, which Compound tooltips do not support as they dynamically figure out their position rather than being provided x,y - this puts it back in the React tree to allow Compound tooltips to work

Reply:
oh I see, gotcha